programs/uxd/Cargo.toml

```diff
@@ -36,10 +36,7 @@ mercurial-vault = { git = "https://github.com/mercurial-finance/vault-sdk", feat
   "cpi",
 ], rev = "3724175d8435f7acb7a33077a5a9f1eb58ab3399" }
 
-# credix_client = { version="0.6.0" } # mainnet
-credix_client = { version = "0.6.0", features = [
-  "pre-mainnet",
-] } # devnet and CI
+credix_client = { version = "0.7.0" }
```

Comment:
this is nice, we now dont need to use a special program ID for only devnet debugging, we can use the same program as the credix team now